src/main/java/com/ridanisaurus/emendatusenigmatica/util/Strata.java

```diff
@@ -10,6 +10,8 @@
 public enum Strata {
 
   STONE("minecraft_stone", "minecraft:block/stone", "stone", Blocks.STONE, "Stone", "minecraft", "overworld"),
+  GRANITE("minecraft_granite", "minecraft:block/granite", "granite", Blocks.GRANITE, "Granite", "minecraft", "overworld"),
+  DIORITE("minecraft_diorite", "minecraft:block/diorite", "diorite", Blocks.DIORITE, "Diorite", "minecraft", "overworld"),
```

Comment:
These two lines and the single line in `mods.toml` are the only real changes.
The rest is all from generated data.